src/main/java/de/embl/cba/mobie/ui/viewer/SourcesPanel.java

```diff
@@ -541,6 +545,7 @@ private void mergeDefaultTableWithAdditionalTables( SourceAndMetadata< ? > sam,
         for ( String tableName : sam.metadata().additionalSegmentTableNames )
         {
             String newTablePath = FileAndUrlUtils.combinePath( tablesLocation, tableName + ".csv" );
+            tableRowsTableView.addAdditionalTables(newTablePath);
```

Comment:
Just a general comment. In `imglib2` the convention is to have a lot of spaces.

For example:

`tableRowsTableView.addAdditionalTables( newTablePath );`

instead of

`tableRowsTableView.addAdditionalTables(newTablePath);`

but "normal" Java would be without the spaces.

`MoBIE` tries to adhere to `imglib2` style. So, if you happen to think about it, it would be nice if you added the spaces. But don't worry now. Maybe one can just tell IntelliJ to make all of this consistent at some point.

Reply:
Ok! Will bear this in mind for the future